tests/snapshots/tests__standard@generic-for-multiline.lua.snap

```diff
@@ -0,0 +1,63 @@
+---
+source: tests/tests.rs
+expression: format(&contents)
+
+---
+local function system(world)
+	for id, model, lasering, transform in world:query(
+		Components.Model,
+		Components.Lasering,
+		Components.Transform,
+		Components.Mothership
+	) do
+	end
+end
+
+local function system(world)
+	for id,
+		model,
+		lasering,
+		transform,
+		id,
+		model,
+		lasering,
+		transform,
+		id,
+		model,
+		lasering,
+		transform,
+		id,
+		model,
+		lasering,
+		transform
+	in world:query(Components.Model, Components.Lasering, Components.Transform, Components.Mothership) do
+	end
+end
+
+local function system(world)
+	for id,
+		model,
+		lasering,
+		transform,
+		id,
+		model,
+		lasering,
+		transform,
+		id,
+		model,
+		lasering,
+		transform,
+		id,
+		model,
+		lasering,
+		transform
+	in world:query(
+		Components.Model,
+		Components.Lasering,
+		Components.Transform,
+		Components.Mothership,
+		Components.Mothership
+	) do
```

Comment:
hm. Is there a reason we don't align the `do` with the `end`, like we do for similar scope closure pairs like: `function()` and `end`, `if` and `else`?

Reply:
In this case, I followed [how prettier did it](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAzTBeTACgEp8A+TYAHSkxwgCcjIoNMBtASwBMAaTALYRucADb9RAQzRwGnKAHN+MBpNbZGAgLqYIuAO6NR3AHQBHAK6yAnoQDCEAQAdoCGGhMBZYWP4PnrrAeADLSsvJKmP4uUG4eACqq6pp+jjFxXhAwABayaNmcTsSk1LR0mCxoEKJwJqIQCoRUIPAYzcQ0dAC+ND1QNJy4hIYMxuZWDLbRge6ZIuJRaTMhYXKKqQGxQSaJamgaDAIb6dveOXkFTsfLmecM+YXXW7NnufeXxZSdmL0gvCAQJwwTjoZCgSQMBgQfQABQhCDQyBAklE+kk1kR-wARqowABrOAwADKkgEcGC8jgyGwKJk2NxBOJTkkYAiyBUVn+cAEWLg3BE3FCigskgUcAAYppJDBgYokZILDAIH8QNkYAJRAB1AqtZlgOBEhGcYEAN2N1iRYDQmJA8hkDBgMNUCgEkmptLg-wAVmgAB5EiI1ACKFiyVKQNNEdJAzPusiRWMkvNEKqcaxgmp4OWQAEYAEwABn+aYgMk1qicSLTcHtJqp-0sYadgMRKGkAFpYny+SqGHBLJw+07Ra73VHPSAZAJOOyGJzJ4G4CGw2PozAk5nuNmkHn-ipJJxRBF-KOUDWAKwqiwyeJJ1uR6MmqwASSgIlgRLAciBAEE30SYGsGpVzgLouiAAl)

It seems that in a for loop, the body opening bracket is aligned with the closing parentheses, but in an if-statement they don't do this...

I think I agree, its probably better to just keep it consistent with how we do `if ... then` expanded multiline. Also makes it easier to handle comments